### PR TITLE
Pmolloy/more mdl things

### PR DESCRIPTION
**Pull request by pamolloy**
- [ ] Add some explanation in the README under `## Contributing`

### CODE REVIEW

**Comment by pamolloy** [comment]
I'm not sure why I didn't mark this as ready to review before, but future me thinks it looks good! :shipit:

File: .github/workflows/markdownlint.yml [hunk 1]
```diff
@@ -13,5 +13,4 @@ jobs:
         run: sudo gem install mdl
 
       - name: Lint markdown
-        continue-on-error: true
-        run: mdl $(find . -name *.md)
+        run: .github/workflows/lazy-lint.bash
```
Comment (orangecms) [inline .github/workflows/markdownlint.yml:16]: Hmm... this no longer invokes `mdl`, only prints the list of files not in the shame list, or am I not understanding it? :grimacing:

Comment (pamolloy) [reply .github/workflows/markdownlint.yml:16]: Not at all! Thanks for noticing. Clearly that is why I didn't mark it as ready for review earlier. :laughing:

**Review by orangecms — APPROVED** [review]

**Comment by pamolloy** [comment]
@orangecms :laughing: it wasn't totally fixed yet because of Bash string quoting and exit code silliness

**Pull request merged**